.gitlab-ci.yml

```
@@ -49,6 +49,7 @@ deploy:
   extends: .base
   stage: deploy
   script:
+    - terraform version | grep -P --color=never '^(\+|Terraform)'
```

```suggestion
    - terraform version
```

is sufficient. I don't mind the out of date warning.